src/containers/shared/components/Transaction/AMMDeposit/Simple.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { SimpleRow } from '../SimpleRow'
+import { TransactionSimpleProps } from '../types'
+import { Account } from '../../Account'
+import { Amount } from '../../Amount'
+
+export const Simple = ({ data }: TransactionSimpleProps) => {
+  const { t } = useTranslation()
+  const { amount, amount2, ammAccountID, ePrice, fee } = data.instructions
+  const value =
+    amount?.amount && amount.amount !== fee ? (
```

Comment:
This doesn't need to be compared to the fee anymore

Reply:
Why do you say that? From what I know it definitely needs to be included.